### DISCUSSION TITLE
Gerei um paladino, saiu um paladino de Lena, mas os poderes estavam um pouco diferentes

### DISCUSSION THREAD

**Discussion by vnmontanhavn** [YuriAlessandro/gerador-ficha-tormenta20#277] [Problemas]
Bem Meu paladino de Lena tava com poderes de Megalokk... 
![image](https://user-images.githubusercontent.com/5113995/114188384-791e5900-991f-11eb-87f1-38be41258dce.png)

**Comment by YuriAlessandro** [comment]
Obrigado pelo feedback. Já conferi aqui no código e está errado mesmo

![image](https://user-images.githubusercontent.com/14967048/114193279-bdf8be80-9924-11eb-9fde-81446ec36ccc.png)

Vou fazer uma correção mais tarde, Está cadastrado como issue #278

**Comment by YuriAlessandro** [comment]
Problema corrigido. Obrigado, mais uma vez!
